labelbox/schema/send_to_annotate_params.py

```diff
@@ -12,21 +12,24 @@
 
 class SendToAnnotateFromCatalogParams(TypedDict):
     """
-    Extra parameters for sending data rows to a project through catalog. At least one of source_model_run_id or
-    source_project_id must be provided.
-
-    :param source_model_run_id: Optional[str] - The model run to use for predictions. Defaults to None.
-    :param predictions_ontology_mapping: Optional[Dict[str, str]] - A mapping of feature schema ids to feature schema
-        ids. Defaults to an empty dictionary.
-    :param source_project_id: Optional[str] - The project to use for predictions. Defaults to None.
-    :param annotations_ontology_mapping: Optional[Dict[str, str]] - A mapping of feature schema ids to feature schema
-        ids. Defaults to an empty dictionary.
+    Extra parameters for sending data rows and their predictions and/or annotations from Catalog to a specified project. 
+    
+    Note: 
+        At least one of source_model_run_id or source_project_id must be provided.
+
+    :param source_model_run_id: Optional[str] - The model run to use for sending predictions to a project. Defaults to None.
+    :param predictions_ontology_mapping: Optional[Dict[str, str]] - A dict of feature schema ids to feature schema
+        ids to map feature schemas from source model run's ontology to target project's ontology.
+        Defaults to an empty dictionary.
```

Comment:
Shouldn't this be required if a source_model_run_id is provided? Is the empty default when a source_model_run_id is not specified? Maybe we should clarify this.

Same comment for `annotations_ontology_mapping`

Reply:
Yeah it's required